ckanext/dcat/converters.py

```diff
@@ -1,4 +1,13 @@
 from past.builtins import basestring
+from ckan.plugins.toolkit import (
+    h,
+    get_action,
+    g,
+    abort,
+    _
+)
+from ckan import model
+from ckan.logic import NotFound
```

Comment:
Please use `toolkit.ObjectNotFound` instead

Reply:
N/A not needed anymore.